### ISSUE TITLE
Clock in analog mode doesn't show date

### ISSUE THREAD

**Issue by jorgenwahlberg** [MichMich/MagicMirror#3100]
When using the clock module and setting displayType to "analog" and showDate to true, date is no longer shown. In earlier releases, it did.

This bug can be reproduced by setting up a the module like this:

```
                {
                        module: "clock",
                        position: "top_left",
                        config: {
                                displayType: "analog",
                                analogSize: "360px",
                                analogPlacement: "bottom",
                                analogFace: "face-003",
                                analogShowDate: "top",
                                showDate: true,
                                displaySeconds: true,
                        }
                }
```

When setting displayType to "both" or "digital", the date is shown.

**Comment by rejas** [comment]
Thanks for the report (and the PR)! Will discuss it in the PR.

**Comment by khassel** [comment]
fixed with new release `v2.24.0`
